File: lógica/JS/estrutura-condicional/multiplos.js
// Fazer um programa para ler dois números inteiros, e dizer se um número é múltiplo do outro. Os
// números podem ser digitados em qualquer ordem.

let a, b;
a = 5;
b = 2;

if(a % b == 2 || b % a == 2) {
    console.log('São multiplos');
} else {
    console.log('Não são múltiplos')
}

File: lógica/Python/estrutura-condicional/multiplos.py
""" Fazer um programa para ler dois números inteiros, e dizer se um número é múltiplo do outro. Os números podem ser digitados em qualquer ordem. """

print("Digite dois números inteiros:")
num1 = int(input())
num2 = int(input())

if num1 % num2 == 0 or num2 % num1 == 0:
    print("São múltiplos")
else:
    print("Não são múltiplos")